src/server/models/db.js

```diff
@@ -0,0 +1,33 @@
+const { MongoClient } = require('mongodb');

```

Comment:
What is the purpose of this file? Is this using MongoDB migration?

Reply:
it has code for connecting to the database, so code doesn't have to be repeated in queries.js